[2024-09-11T02:38:18.247+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T02:38:18.302+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-07-30T00:00:00+00:00 [queued]>
[2024-09-11T02:38:18.319+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-07-30T00:00:00+00:00 [queued]>
[2024-09-11T02:38:18.320+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T02:38:18.376+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2024-07-30 00:00:00+00:00
[2024-09-11T02:38:18.386+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=18272) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T02:38:18.386+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2024-07-30T00:00:00+00:00', '--job-id', '4028', '--raw', '--subdir', 'DAGS_FOLDER/13_dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmppb1ujzxv']
[2024-09-11T02:38:18.388+0000] {standard_task_runner.py:63} INFO - Started process 18278 to run task
[2024-09-11T02:38:18.388+0000] {standard_task_runner.py:91} INFO - Job 4028: Subtask delete_data_from_table
[2024-09-11T02:38:18.456+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2024-07-30T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T02:38:18.580+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-30T00:00:00+00:00'
[2024-09-11T02:38:18.581+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T02:38:18.609+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2024-07-30' and dag_id = 'dag_with_postgres_operator_v03'
        
[2024-09-11T02:38:18.623+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:18.640+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:18.658+0000] {sql.py:487} INFO - Running statement: 
            delete from dag_runs where dt = '2024-07-30' and dag_id = 'dag_with_postgres_operator_v03'
        , parameters: None
[2024-09-11T02:38:18.663+0000] {sql.py:496} INFO - Rows affected: 0
[2024-09-11T02:38:18.666+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T02:38:18.684+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, run_id=scheduled__2024-07-30T00:00:00+00:00, execution_date=20240730T000000, start_date=20240911T023818, end_date=20240911T023818
[2024-09-11T02:38:18.725+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T02:38:18.736+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
